themes/bootstrap3/scss/components/search.scss

```diff
@@ -864,3 +864,10 @@ table.search-history-table {
     }
   }
 }
+
+.backtoorigin > p > a {
```

Comment:
Since there are less/scss changes in this PR, the compiled.css assets should be committed as well.